openwisp_network_topology/management/commands/upgrade_from_django_netjsongraph.py

```diff
@@ -0,0 +1,172 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from swapper import get_model_name, load_model, split
+
+User = get_user_model()
+Topology = load_model('topology', 'Topology')
+Organization = load_model('openwisp_users', 'Organization')
+
+
+class BaseUpdateFromDjangoNetjsonGraph(BaseCommand):
+    help = 'Upgrade from django-netjsongraph'
+    app_label_users = split(get_model_name('openwisp_users', 'Organization'))[0]
+    app_label = split(get_model_name('topology', 'Topology'))[0]
+
+    def add_arguments(self, parser):
+        dirname = os.path.join(
+            # Path to `tests/` directory
+            os.path.split(os.path.abspath(__file__))[0],
+            '..',
+            '..',
+            '..',
+            'tests',
+        )
+        parser.add_argument(
+            '--backup',
+            action='store',
+            default=dirname,
+            help='(Optional) Path to the backup files',
+        )
+        parser.add_argument(
+            '--organization',
+            action='store',
+            default=None,
+            help=(
+                '(Optional) organization UUID of the organization in '
+                'which you want to import the data.'
+            ),
+        )
+
+    def int_to_uuid(self, pk):
+        return "00000000-0000-0000-0000-{:012}".format(pk)
+
+    def handle(self, *args, **options):
+        if options['organization']:
+            org = Organization.objects.get(pk=options['organization'])
+        else:
+            org = Organization.objects.first()
+
+        # netjsongraph.json
+        with open(f'{options["backup"]}/netjsongraph.json') as netjsongraph:
+            netjsongraph_data = json.load(netjsongraph)
+        # Make changes
+        for data in netjsongraph_data:
+            data['fields']['organization'] = str(org.pk)
+            data['model'] = f'{self.app_label}.{data["model"].split(".")[1]}'
+        # Save in anotherfile
+        with open(f'{options["backup"]}/netjsongraph_loaded.json', 'w') as outfile:
+            json.dump(netjsongraph_data, outfile)
+        # Load to database
+        call_command(
+            'loaddata', f'{options["backup"]}/netjsongraph_loaded.json', verbosity=0
+        )
+
+        # group.json & permission
+        with open(f'{options["backup"]}/permission.json') as permission:
+            permission_data = json.load(permission)
+        with open(f'{options["backup"]}/group.json') as group:
+            group_data = json.load(group)
+        load_group_data = []
+        for data in group_data:
+            permit_list = []
+            for permit_pk in data['fields']['permissions']:
+                for item in permission_data:
+                    if item['pk'] == permit_pk:
+                        permit_list.append(
+                            Permission.objects.filter(
+                                codename=item['fields']['codename']
+                            )
```

Comment:
I think it would be more robust doing something like:

```
Permission.objects.get(content_type__app_label='...', codename=item['fields']['codename'])
```